### PR TITLE
Bugfix/s3 c 4139 allow only start timestamp in req

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

**Review by naren-rajendran — APPROVED** [review]

File: tests/functional/v2/server/testListMetrics.js [hunk 1]
```diff
@@ -30,10 +30,15 @@ const emptyOperationsResponse = Object.values(operationToResponse)
...
     const body = {
-        timeRange: [start, end],
         [level]: resources,
     };
 
+    if (end !== undefined) {
+        body.timeRange = [start, end];
+    } else {
+        body.timeRange = [start];
+    }
```
Comment (miniscruff) [inline tests/functional/v2/server/testListMetrics.js:40]: I think `[start, end]` works in both cases as end would be undefined.

**Review by miniscruff — APPROVED** [review]

**Review by alexanderchan-scality — APPROVED** [review]

File: tests/functional/v2/server/testListMetrics.js (hunk 1 above)
Comment (tmacro) [reply tests/functional/v2/server/testListMetrics.js:40]: it serializes to `[0, null]` for some reason if you leave it in

**Pull request merged**